src/components/course/data/hooks.jsx

```diff
@@ -51,12 +52,19 @@ export function useAllCourseData({ courseKey, enterpriseConfig, courseRunKey })
           logError(error);
           setFetchError(error);
         }
+
+        try {
+          const data = await courseService.fetchAllCourseRecommendations();
+          setCourseRecommendations(data);
+        } catch (error) {
+          logError(error);
+        }
       }
       return undefined;
     };
     fetchData();
   }, [courseKey, enterpriseConfig]);
-  return [camelCaseObject(courseData), fetchError];
+  return [camelCaseObject(courseData), camelCaseObject(courseRecommendations), fetchError];
```

Comment:
what if both of the above api calls in this hook fails, the `fetchError` would contain the error for last call, first would be lost.


Should we send second call if first fails?
if we should then should we keep both errors separate ?

Reply:
In case if the first api call succeeds and second doesn't, we'd still want to show the course content instead of crashing the whole page to throw the error. So we are only logging the error if second API fails.